### DISCUSSION TITLE
Syntax highlighting for ANSI escape code

### DISCUSSION THREAD

**Discussion by Sammie26559** [stoatchat/discussions#468] [Feature Suggestions]
I'm the only one who thinks it would be a good idea to have ANSI escape code support for syntax highlighting (in code blocks) like on discord:

how to write it on discord:
![](https://user-images.githubusercontent.com/47297859/205461858-57bf5f51-ece4-42b3-b318-7917b39e04b8.png)
how does it look like once the message is sent:
![ ](https://user-images.githubusercontent.com/47297859/205462310-a9a6e79a-f39d-4d47-964b-5d22591253c8.png)

ANSI escape code full doc in wikipedia : https://en.wikipedia.org/wiki/ANSI_escape_code

**Comment by Rexogamer** [comment]
Duplicate of revoltchat/revite#782.

**Reply by Sammie26559** [reply]
Sorry I didn't see it.
